package.json

```diff
@@ -33,6 +33,8 @@
     "watch:lessToSass": "grunt watch:lessToSass"
   },
   "dependencies": {
+    "@splidejs/splide": "^4.1.4",
```

Comment:
These files now belong in https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap3/package.json and should be removed from this top-level file.